core/src/tests/parachain.rs

```diff
@@ -601,22 +597,22 @@ async fn multi_parachains_can_send_xcm_messages() {
 		remark: remark.clone(),
 	});
 
-	let para_1_location = VersionedMultiLocation::from(MultiLocation::new(
+	let para_1_location = VersionedLocation::from(Location::new(
 		1,
-		Junctions::X1(Junction::Parachain(PARA_ID_1)),
+		Junctions::X1(Arc::new([Junction::Parachain(PARA_ID_1)])),
```

Comment:
New XCM v4 crates allow to create X1 just like this
```suggestion
		Junction::Parachain(PARA_ID_1),
```